org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -25,7 +25,16 @@ modules:
         sha512: 5b95a63e77953c540ce1b8e6338d7189785f19333a34c73ae97662d5691968ccaaff9bfd712d2006ebec518a6a6545c00adaf97afd22bfc1fa35ab47c8ce4474
         dest-filename: texlive-texmf.tar.xz
   - name: luametatex
-    buildsystem: cmake
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
```

Comment:
is there a reason to not put them directly in `bin` ? there is only architecture in the flatpak.

Reply:
bit this make it the prefix which is likely not what you want.

Reply:
Sorry, I did not get your comments in time. I tried to reproduce the instructions of https://tug.org/pipermail/tex-live/2023-March/048960.html but it took some time.